Formula/kamel.rb

```diff
@@ -0,0 +1,54 @@
+class Kamel < Formula
+  desc "Apache Camel K CLI"
+  homepage "https://camel.apache.org/"
+
+  url "https://github.com/apache/camel-k.git",
+    :tag      => "0.3.4",
+    :revision => "c47fb2c85e89852f0fd111d1662f57917030ced5"
+  head "https://github.com/apache/camel-k.git"
+
+  depends_on "go" => :build
+  depends_on "openjdk"
```

Comment:
Isn't this build only as well?

Reply:
@SMillerDev apologies I got a bit confused, could you please elaborate more?

Reply:
Isn't the jdk just used when building the software? Or is it also used at runtime?

Reply:
it is also used in the [camel-k-runtime](https://github.com/apache/camel-k-runtime).

Reply:
@SMillerDev now I got it, you are right, _openjdk_ should also be declared as build only. There is no dependency of the CLI to this on runtime.

 I have made the change and rebased. If you could please verify.